pkg/build/local/command.go

```diff
@@ -45,6 +47,9 @@ func (r *realCommandExecutor) Execute(ctx context.Context, opts CommandOptions,
 		cmd.Stdout = opts.Output
 		cmd.Stderr = opts.Output
 	}
+	if opts.Directory != "" {
```

Comment:
I believe you should be able to simplify this to unconditionally set this property